tools/utils/getBranchName.js

```diff
@@ -0,0 +1,11 @@
+const {exec} = require('child_process');
+
+module.exports = (callback) => {
```

Comment:
Super ruff around the edges script to get the branch name in a local env or if git is defined... pretty specifically intended for the cypress/eyes integration. Curious what folks think of the basic principle here, of:
1. figuring out the branch name 
2. using the branch name and the environment to name applitools branches